ReadMe.md

```diff
@@ -5,7 +5,8 @@ Benchmark automatic language boxes in Eco.
 ## Requirements
 
 - Python2
-- (Optional) PyPy
+- lib32z1
+- (Optional) PyPy (greatly improves runtime)
```

Comment:
I think this means "(makes the experiment run much faster)"?

Reply:
Fixed.